playbook/app/pb_kits/playbook/pb_selectable_icon/_selectable_icon.scss

```diff
@@ -5,6 +5,7 @@
 	text-align: center;
 	cursor: pointer;
 	transition: all $transition_short ease;
+
```

Comment:
Looks like your linter/formatter is doing some wonky indentation. I don't mind the blank lines here if they are intentional for making some space/breathing room, ....but i would want you to investigate later getting prettier configured locally or something.